datalad/__init__.py

```diff
@@ -92,6 +92,8 @@ def test(module='datalad', verbose=False, nocapture=False, pdb=False, stop=False
     'HOME': None,
 }
 
+# handle to an HTTP server instance that is used as part of the tests
+test_http_server = None
```

Comment:
why not to make it at least `_test_http_server` or even better just place within `_test_states['http_server']` - I see no value in polluting top level namespace

Reply:
Yes, I agree, we can tune that.